inference/app/__init__.py

```diff
@@ -36,19 +37,19 @@ def allowed_filesize(filesize):
 @app.route("/upload_file", methods=["POST"])
 def uploadfile():
     if request.method == 'POST':
-        if request.files:
-            if "filesize" in request.cookies:
-                if not allowed_filesize(request.cookies["filesize"]):
-                    return handle_bad_request()
-            else:
-                f1 = request.files["file"]
-                if allowed_file(f1.filename):
-                    full_filename = os.path.join(
-                        app.config['UPLOAD_FOLDER'],
-                        secure_filename(
-                            f1.filename))
-                    f1.save(full_filename)
-                    return make_response(jsonify(request.files), 200)
-                else:
-                    return handle_bad_request()
-    return make_response(jsonify(request.files), 200)
+        if "filesize" in request.cookies:
+            if not allowed_filesize(request.cookies["filesize"]):
+                return handle_bad_request('e')
+        f1 = request.files["file"]
+        if allowed_file(f1.filename):
+            full_filename = os.path.join(
+                app.config['UPLOAD_FOLDER'],
+                secure_filename(
+                    f1.filename))
+            f1.save(full_filename)
```

Comment:
can this be loaded to mongo directly without writing to disk?

Reply:
something like mongofiles ?

Reply:
no just inserting the JSON into mongo directly using something like `insert_many` http://api.mongodb.com/python/current/api/pymongo/collection.html#pymongo.collection.Collection.insert_many?

Reply:
Isn't this the case of multiple files? Can we discuss this in a newer issue as this one already has two of them

Reply:
Maybe newer issue ?